packages/evm/contracts/test/arbitration/Arbitrable.sol

```diff
@@ -22,8 +22,15 @@ contract ArbitrableMock is IArbitrable {
         if (_finished) arbitrator.closeEvidencePeriod(_disputeId);
     }
 
+    function submitEvidenceFor(uint256 _disputeId, address _submitter, bytes calldata _evidence, bool _finished) external {
```

Comment:
I couldn't find an instance of this being used yet; was it meant to be used in a test?

Reply:
This was actually being used in the court backend CLI to deploy a mock arbitrable